Control Plane Lite documentation update (#915)
rudderlabs/rudderstack-docs: docs/get-started/rudderstack-open-source/control-plane-lite.mdx

---

RudderStack's <Link to="/resources/rudderstack-architecture/#control-plane">control plane</Link> provides a UI that lets you manage your source and destination configurations.

<div class="successBlock">
The easiest way to manage your configurations is through <a href="https://app.rudderstack.com/signup">RudderStack Cloud</a>. It is free, requires no setup, and has some useful features like <Link to="/dashboard-guides/live-events/">Live Events</Link> and <Link to="/features/transformations/">Transformations</Link>.
</div>

To self-host your source-destination configurations, you can use the open-source <a href="https://github.com/rudderlabs/config-generator">Control Plane Lite</a> utility to set up your own control plane. It lets you manage your data pipelines locally by exporting or importing your configurations from a JSON file.

<div class="dangerBlock">
<a href="https://github.com/rudderlabs/config-generator">Control Plane Lite</a> is no longer actively maintained. As a result, some of the <a href="https://app.rudderstack.com/signup">RudderStack Cloud</a> features may not be available in your self-hosted control plane.
</div>

## Setting up the control plane

> Before setting up your self-hosted control plane, make sure you have installed <a href="https://nodejs.org/en/download/">Node.js</a>.

Reviewer: ```suggestion
Before setting up your self-hosted control plane, make sure you have installed <a href="https://nodejs.org/en/blog/release/v14.17.3/">Node.js</a>.
```
As downloading this version might not work

<div class="warningBlock">

> Node.js v16 and higher are not supported. For the best experience, using Node.js v14 is strongly recommended.

Reviewer: ```suggestion
Use Node.js v14 for the best experience as the higher versions are not supported. 
```

Once you have completed these steps above successfully, <Link to="/get-started/rudderstack-open-source/sending-test-events/">send test events</Link> to verify the installation.

> ### Control plane URL

Reviewer: ```suggestion
## Control plane URL
```

## FAQ

> ### How do I obtain the control plane URL for a self-hosted environment?

Reviewer: can skip as same steps are present in the above section

## FAQ

### How do I obtain the control plane URL for a self-hosted environment?

> <div class="infoBlock">

Author: ```suggestion
```

> This solution assumes that you have already set up the RudderStack data plane (backend) locally.

Reviewer 2: Is this relevant to the question? Data plane setup is not a precondition for obtaining the CP url

Author: Makes sense. I will remove this note.

> This solution assumes that you have already set up the RudderStack data plane (backend) locally.

Author: ```suggestion
```

> </div>

Author: ```suggestion
```
